executor/explain_test.go

```diff
@@ -260,7 +260,7 @@ func (s *testSuite) TestExplain(c *C) {
     "ranges": "[[1,1]]",
     "desc": false,
     "out of order": true,
-    "double read": false,
+    "double read": true,
```

Comment:
double read first read handle from index request, then use handle to send table request to read full row.